lib/createTodobox.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function createTodobox(todobox){
```

Comment:
createTodobox function attached to module export to allow for todobox to pass through